reconciletags/reconciletags.py

```diff
@@ -52,12 +52,17 @@ def reconcile_tags(self, data, dry_run):
         # reconciler.
         self.call('gcloud config list', False)
         for project in data['projects']:
-            for registry in project['registries']:
+            default_registry = project['base_registry']
+            registries = project['additional_registries']
```

Comment:
Maybe make this a ".get" check in case someone leaves it off. This field should be optional.